src/fragments/lib/datastore/flutter/data-access/observe-update-snippet.mdx

```diff
@@ -0,0 +1,55 @@
+```dart
+void main() {
+  runApp(MyApp());
+}
+
+class MyApp extends StatefulWidget {
+  const MyApp({Key? key}) : super(key: key);
+
+  @override
+  State<MyApp> createState() => _MyAppState();
+}
+
+class _MyAppState extends State<MyApp> {
+  StreamSubscription<QuerySnapshot<Post>>? _stream;
+
+  // A place to keep the post we care about
+  Post? _post;
```

Comment:
If we want to leverage from the _isFirstSync kind of variable, then we can turn this into a non-nullable type with late keyword.